lessonbuilder/tool/src/webapp/js/announcements.js

```diff
@@ -40,7 +40,7 @@ function showAnnouncements(url, tool_href, number, announcementsDiv){
 						var href = tool_href + this["announcementId"]+"&sakai_action=doShowmetadata&persist_to_iframe=itemReference";
 						var entityTitle = this["entityTitle"].replace(/&/g, '&amp;').replace(/>/g, '&gt;').replace(/</g, '&lt;').replace(/"/g, '&quot;');
 						var createdByDisplayName = this["createdByDisplayName"].replace(/&/g, '&amp;').replace(/>/g, '&gt;').replace(/</g, '&lt;').replace(/"/g, '&quot;');
-						text_for_announcements += '<div><a href="'+href+'" target="_top">'+ entityTitle +'</a> by '+ createdByDisplayName +'</div>';
+						text_for_announcements += '<div>'+ (this["highlight"]=="true" ? '<i aria-hidden="true"class="fa fa-star"></i>' : '') + '<a ' + (this["highlight"]=="true" ? 'class="font-bold-highlighted"' : '') + ' href="'+href+'" target="_top">' + entityTitle +'</a> by '+ createdByDisplayName +'</div>';
```

Comment:
I feel this is complex to read but I understant that JS file has many many bad patterns like this, and I also see existing L10N issues.

Look at this, but don't implement it, have it in mind for the next time:

`
const isHighlighted = "true" === this["highlight"];
let highlightClass = '';
text_for_announcements += '<div>';
if (isHighlighted) {
  text_for_announcements += '<i aria-hidden="true" class="fa fa-star"></i>';
  highlightClass = 'font-bold-highlighted';
}
text_for_announcements += `<a href="${href}" class="${highlightClass}" target="_top">${entityTitle}</a> by ${createdByDisplayName}</div>`;
`

Reply:
Ok. For the next time i will do something similar to this. Because it is more simple to read

Reply:
Yeah, template strings are way better than this.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals